### ISSUE TITLE
Some advice

### ISSUE THREAD

**Issue by Yukimmu** [AmberYYYYYYYY/Cuisine-Filter#1]
Hi AmberYYYYYYYY，

I have tested your tool. 
![Picture](https://user-images.githubusercontent.com/88954942/132277628-7aae67f2-6563-4ffd-b30a-db2e6d2dcbc5.jpg)
I think it is very good. People can use it to choose some suitable sites for eating, but if people want more information, for example, he wants to go to the restaurant closer to him,  or he is a local resident, he wants to go to a restaurant closer to home, or he is a tourist, he wants to go to the restaurant closer to the hotel, how should he choose? Maybe You can add more details to your tool.

**Comment by AmberYYYYYYYY** [comment]
Hi Yuki，
Thanks a lot for your great suggestion! Yes! add a navigation function is a good idea, and I just add the shortest distance technique into my file to provide route suggestions for users but it doesn't work good :(   Hopefully, I could fix it this week and I will let you know once I get this done.

**Comment by AmberYYYYYYYY** [comment]
Hi, I have added the navigation function in my file, just a simple one: help users find the closest food court from your location and count the distance. I also developed an advanced version of the shortest walk path, but my file size brought some trouble to split curves.  I hope I can get this down soon. 

Here is the newest version:  https://github.com/AmberYYYYYYYY/Test-mappingtool 
I am looking forward to your feedback.
